### PR TITLE
[CLASSIC-20568] extracting PDF417 and mavenizing sources

### PR DESCRIPTION
**Pull request by assentis-wsr**
Sources reduced to PDF417 support only and mavenized.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    jdkVersionOption: '1.8'
+    jdkArchitectureOption: 'x64'
+    publishJUnitResults: false
+    testResultsFiles: '**/TEST-*.xml'
+    goals: 'deploy'
+    mavenFeedAuthenticate: true
+    options: '-B -Drevision=$(patchBaseVersion)'
+    codeCoverageToolOption: 'None'
+    sonarQubeRunAnalysis: false
+  condition: eq(variables['Build.SourceBranch'])
```
Comment (mnowrot) [inline azure-pipelines.yml]:
```suggestion
  condition: eq(variables['Build.SourceBranch'], 'master')
```
or is it by default?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    sonarQubeRunAnalysis: false
+  condition: and(not(startsWith(variables['Build.SourceBranch'], 'refs/pull')),ne(variables['Build.SourceBranch'], 'master'))
+
+  # master build is doing a release (no SNAPSHOT set). This requires to properly update the version
+  # in this pipeline step each time one wants to do a release...
+- task: Maven@3
+  name: 'Release_Task'
+  inputs:
+    mavenPomFile: 'pom.xml'
+    mavenOptions: '-Xmx3072m'
```
Comment (mnowrot) [inline azure-pipelines.yml:37]: I'd remove it and check if it still runs - we probably don't need that much, this was once set for DF builds

Comment (mnowrot) [reply azure-pipelines.yml:37]: (applies to all taks)

**Review by mnowrot — APPROVED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (assentis-wsr) [reply azure-pipelines.yml:37]: you are right, copied from DF build

**Pull request merged**